common-ui/src/main/java/org/simple/clinic/common/ui/util/TextUtil.kt

```diff
@@ -0,0 +1,36 @@
+package org.simple.clinic.common.ui.util
+
+import android.graphics.Typeface
+import android.os.Build
+import android.text.Html
+import android.text.Spanned
+import android.text.style.StyleSpan
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.text.font.FontWeight
+
+fun htmlToAnnotatedString(htmlText: String) = getSpannedText(htmlText).toAnnotatedString()
+
+private fun Spanned.toAnnotatedString(): AnnotatedString = buildAnnotatedString {
```

Comment:
We can just have a single extension function, no? Something like this

```kt
fun String.toAnnotatedString(): AnnotatedString {
  val spanned = getSpannedText(this)
  ...
}
```

That way we can just call `stringResources(...).toAnnotatedString()`. WDYT?